src/llamacrawl/services/readers/firecrawl.py

```
@@ -510,6 +625,31 @@ async def _aload_single_url(
             **kwargs,
         )
 
+    def _extract_job_id(self, response: Any) -> str:
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `response`

(ANN401)